File: src/entity/enemy/BossEnemy.java
package entity.enemy;

import javafx.scene.image.Image;

public class BossEnemy extends Enemy {
	
	public static final String IMAGE_URL = "file:img/PNG/Retina/towerDefense_tile246.png";
	
	public BossEnemy(int x, int y) {
		super(x, y, 20, 2.5);
		img = new Image(IMAGE_URL);
		setDir('E');
	}
	
}

File: src/entity/enemy/BossEnemy.kt
package entity.enemy

import javafx.scene.image.Image

class BossEnemy(x: Int, y: Int) : Enemy(x.toDouble(), y.toDouble(), 20, 2.5) {
    override var img: Image = Image(IMAGE_URL)

    init {
        setDir('E')
        reward = 10
    }

    companion object {

        val IMAGE_URL = "file:img/PNG/Retina/towerDefense_tile246.png"
    }

}
